### DISCUSSION TITLE
hard limit for the projects

### DISCUSSION THREAD

**Discussion by ctr0603** [goharbor/harbor#17384] [Ideas]
Hi Team,

Is it possible to allocate space for each project and set up a hard limit project wise
For example i have projectA projectB and projectC

Off 500GB Space i allocated 100GB for projectA 100GB for projectB and 100GB for ProjectC

projectA -- 100GB space allocated and i want to set 80GB Hardlimt so that as soon as 80GB is reached i will not be able  to push images  and i have to clean some of the images to do that

similarly for projectB and projectC

**Comment by dioguerra** [comment]
Having space allocated and a hard limit is pretty much the same. Use quota for this.
